recipes/phyde/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "phyde" %}
+{% set version = "0.3.2" %}
+{% set sha256 = "83bae73906205e219a528c523ec6a168fb7dc9cc0116a8e6c97e1ba32a7c9f9d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win or (not py27)]
+  number: 2
```

Comment:
You can set this to 0 in staged-recipes

Reply:
Gotcha -- will do. Thanks for letting me know!